src/scripts/main.js

```diff
@@ -1,3 +1,65 @@
 'use strict';
 
-// write code here
+const queryList = document.querySelector('ul').children;
+
+const getEmployees = function(list) {
+  const arrayList = [...list].map(item => {
```

Comment:
```suggestion
  const arrayList = list.map(item => {
```

Reply:
I can't do that, list is a collection, there seems to be no way to use "map"